packages/yew/src/dom_bundle/braw.rs

```diff
@@ -0,0 +1,220 @@
+use wasm_bindgen::JsCast;
+use web_sys::Element;
+
+use crate::dom_bundle::bnode::BNode;
+use crate::dom_bundle::traits::{Reconcilable, ReconcileTarget};
+use crate::dom_bundle::utils::insert_node;
+use crate::dom_bundle::BSubtree;
+use crate::html::AnyScope;
+use crate::virtual_dom::VRaw;
+use crate::NodeRef;
+
+#[derive(Debug)]
+pub struct BRaw {
+    reference: NodeRef,
+    children_count: usize,
+}
+
+impl BRaw {
+    fn create_elements(html: &str) -> Vec<Element> {
+        let div = gloo::utils::document().create_element("div").unwrap();
+        let html = html.trim();
+        div.set_inner_html(html);
+        let children = div.children();
+        let children = js_sys::Array::from(&children);
+        let children = children.to_vec();
+        children
+            .into_iter()
+            .map(|it| it.unchecked_into())
+            .collect::<Vec<_>>()
+    }
+
+    fn detach_bundle(&self, parent: &Element) {
+        if let Some(node) = self.reference.cast::<Element>() {
```

Comment:
What's the reason to cast to an `Element`? Both `next_sibling` and `remove_child` seem to work for `Node`, iirc.

Reply:
None, I fixed it